core/unix/loader.c

```diff
@@ -82,6 +82,11 @@ static const char *const system_lib_paths[] = {
     "/usr/local/lib", /* Ubuntu: /etc/ld.so.conf.d/libc.conf */
 #ifdef ANDROID
     "/system/lib",
+#    ifdef ANDROID64
+    "/system/lib64",
+#    elif defined(ANDROID32)
+    "/system/lib32",
+#    endif
```

Comment:
Added a comment, given people seemed to have some success running DR with that PR it would probably make sense to keep it alive until support for _running_ with the new Clang NDK is added?